### ISSUE TITLE
about dense small objects detection

### ISSUE THREAD

**Issue by 791136190** [Megvii-BaseDetection/YOLOX#804]
I implemented the label matching and training functions with reference to the training code you provided, but there are some problems in the detection of some dense small targets.
The following are the results of normal detection and error detection.
![image](https://user-images.githubusercontent.com/27891060/137831986-a75d4f13-78d0-4344-960c-c38cdca739e2.png)
![image](https://user-images.githubusercontent.com/27891060/137832086-186d28a1-0222-436b-a8f4-cb14be23a1fe.png)
![image](https://user-images.githubusercontent.com/27891060/137832152-f99aa27c-61d1-49e7-80ba-1ccafab85a49.png)
As you can see, there are some very large error rectangles in the detection results shown in the next two pictures.Such wrong boxes are all output on the deepest detection layer.
I want to confirm whether there will be such a problem in the official code.Could this be an OTA problem？

**Comment by Joker316701882** [comment]
@791136190 As @ruinmessi said, remember to remove the ground truth with the ''crowd'' flag.

**Comment by 791136190** [comment]
@ruinmessi @Joker316701882 Yes, my dataset does not include the “crowd attribute”, and when I use the coco dataset, I also removed the “crowd” label. It may be because some data classification predictions are correct, but regression predictions are wrong.

**Comment by BatmanofZuhandArrgh** [comment]
seems like a simple confidence threshold should be able to remove most of the big boxes. Weird, cuz I also trained on far faces, and did not experience weird big boxes like those
